modules/markup/csv/csv.go

```diff
@@ -174,6 +128,22 @@ func (Renderer) tableRender(ctx *markup.RenderContext, input io.Reader, tmpBlock
 
 		row++
 	}
+
+	// Check if maxRows or maxSize is reached, and if true, warn.
+	if (row >= maxRows && maxRows != 0) || (rd.InputOffset() >= maxSize && maxSize != 0) {
+		locale := ctx.Ctx.Value(translation.ContextKey).(translation.Locale)
+
+		// Construct the HTML string
+		warn := `<div class="ui top attached warning message" tabindex="0">` + locale.TrString("repo.file_too_large") + ` <b><a class="source" href="?display=source">` + locale.TrString("repo.file_view_source") + `</a></b></div>`
```

Comment:
```suggestion
		warn := `<div class="tw-flex tw-justify-center tw-items-center"><div>` + locale.TrString("repo.file_too_large") + ` <a class="source" href="?display=source">` + locale.TrString("repo.file_view_source") + `</a></div></div>`
```

Warning message is too flashy and does not fit visually inside this table at all. Let's try something more subtle, similar to diffs that are too large.

Reply:
Yes, looks good!